modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1172,8 +1179,8 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
           );
           continue;
         }
-        const privKey = keychain.derivePath(signatureContext.path).keyPair;
-        privKey.network = self.network;
+
+        const keyPair = keychain.derivePath(sanitizeLegacyPath(signatureContext.path));
```

Comment:
How come we're assigning `network` to the `keychain` now instead of the `keyPair` (what used to be called `privKey`)?

Do we need the network property on the `keychain` at all?

Reply:
I am now assigning `network` in L1147 already

Currently `txb.network` needs to match `keychain.network` which is `networks.bitcoin` by default.